### PR TITLE
Feat/Custom device info

### PR DESCRIPTION
**Pull request by ogustavobelo**
Allow creating devices preview with custom sizes and platforms 

https://user-images.githubusercontent.com/53406077/148651642-2c23c88e-7dfa-44d1-8e51-152eff7fcef9.mp4

### CODE REVIEW

**Comment by spydon** [comment]
Nice! Looks very good, you should be able to attach the video directly to the PR description.

**Comment by ogustavobelo** [comment]
> Nice! Looks very good, you should be able to attach the video directly to the PR description.

Nice, It is atached now

**Review by spydon — APPROVED** [review]
LGTM!

File: lib/src/widgets/select_device/custom_device.dart [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    required this.label,
+    required this.onUpdate,
+    Key? key,
+  }) : super(key: key);
+  final TextEditingController controller;
+  final String label;
+  final void Function(num) onUpdate;
+
+  String? _numValidator(String? value) {
+    if (value == null || value.isEmpty) return 'Value could not be empty';
```
Comment (erickzanardo) [inline lib/src/widgets/select_device/custom_device.dart:151]:
```suggestion
    if (value == null || value.isEmpty) return 'Value can not be empty';
```

**Review by tyemykuga — APPROVED** [review]

**Pull request merged**